Gemfile

```diff
@@ -1,5 +1,7 @@
 source "https://rubygems.org"
 
+ruby '2.7.0'
```

Comment:
Just remove that line and it should work on Travis AFAICS.

Reply:
as long as travis defaults to ruby 2.5+ it should work. force pushed.